priv/priv_test.go

```diff
@@ -53,11 +53,11 @@ func (ts *PrivTestSuite) TestPrivMutex(c *C) {
 	c.Assert(helpers.FileExists(lockfile), Equals, false)
 
 	privMutex := New(lockfile)
-	c.Assert(privMutex, Not(IsNil))
+	c.Assert(privMutex, NotNil)
 	c.Assert(helpers.FileExists(lockfile), Equals, false)
 
 	err := privMutex.Unlock()
-	c.Assert(err, DeepEquals, ErrNotLocked)
+	c.Assert(err, NotNil)
```

Comment:
sys.EBADF
Not sure ErrorMatches makes sense -- literally, not sure. What do you think?

On 2 November 2015 at 12:27, Michael Vogt <notifications@github.com> wrote:

> In priv/priv_test.go
> <https://github.com/ubuntu-core/snappy/pull/42#discussion_r43623161>:
>
> >  	c.Assert(helpers.FileExists(lockfile), Equals, false)
> >
> >  	err := privMutex.Unlock()
> > -	c.Assert(err, DeepEquals, ErrNotLocked)
> > +	c.Assert(err, NotNil)
>
> What kind of error do we get now? Would it make sense to user
> "ErrorMatches" in this test instead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ubuntu-core/snappy/pull/42/files#r43623161>.
>